requirements/base.txt

```diff
@@ -1,8 +1,9 @@
 # Requirements for app run
 
-git+https://github.com/edx/xblock-utils.git@v1.1.1#egg=xblock-utils==1.1.1
-django-statici18n==1.8.0
-edx-i18n-tools==0.4.7
-Mako==1.0.7
-bleach==2.1.4
-html-xblock~=0.1.1
+git+https://github.com/edx/xblock-utils.git@2.1.1#egg=xblock-utils==2.1.1
```

Comment:
@taniwha, the `Makefile` has the `requirements` target set up in a weird way where the `test`, `quality` requirements are installed first before the `base` requirements and this causes some warnings to be thrown about a newer version of Django being installed than one of the dependencies expects and it eventually gets fixed later when `Django~=2.2` gets installed. We can fix this by fixing the `Makefile` targets or by adding a `-r base.txt` at the top of the `test`, `quality` requirements and updating the `Makefile` targets to take that into account. What do you say?

The same issue is present even on `xblock-html` and whatever we choose to do here should be done there as well.

Reply:
To maintain the apparent intent, I think adding a base_requirements rule would take care of things cleanly, but I agree with the basis of your suggestion.